src/index.js

```diff
@@ -3351,8 +3369,8 @@ class ULabel {
                 annotation_mode: annotation_mode,
                 gmx: gmx,
                 gmy: gmy,
-                init_spatial: init_spatial,
-                finished: redoing || annotation_mode == "polygon",
+                init_spatial: JSON.parse(JSON.stringify(init_spatial)),
```

Comment:
what does the stringify->parse accomplish?

Reply:
It's the only way I know of in order to make a deep copy